src/header/header.jsx

```diff
@@ -23,6 +23,9 @@ function Header() {
         </div>
         <div className="top_padding_burger">
           <img width="25" height="16" src="/photos/burger.png" alt="burger" />
+          <div className="hoover_menu">
```

Comment:
```suggestion
          <div className="hover_menu">
```
check spelling in your code